nameserver/entry.go

```diff
@@ -190,8 +194,7 @@ func (es *Entries) tombstone(ourname mesh.PeerName, f func(*Entry) bool) Entries
 
 	tombstoned := Entries{}
 	for i, e := range *es {
-		if f(&e) && e.Origin == ourname {
-			e.tombstone()
+		if f(&e) && e.Origin == ourname && e.tombstone() {
```

Comment:
Slight mis-match between the logging and the behaviour now: the sub-functions of `ContainerDied()` and `Delete()` will log "tombstoning entry" when it is already tombstoned.